### ISSUE TITLE
When exporting functions from a C++ module, it is necessary to specify EXPORTED_FUNCTIONS for each function.

### ISSUE THREAD

**Issue by xengine-qyt** [emscripten-core/emsdk#1452]
Is there any other way to automatically export all functions from a file?

**Comment by sbc100** [comment]
No, there is no mechanism for that I'm afraid.    I'm not sure I'd want to add one either.  What would your use case be fore exports so many functions?   Normally folks either exports small fixed set of functions, or use something like embind to export a more rich set.   What is your use case?
